### ISSUE TITLE
Bad Gateway 502

### ISSUE THREAD

**Issue by kocipepek** [hassio-addons/addon-grocy#234]
First of all, sorry for my poor english :(

I`ve already install grocy but when I start UI of Grocy I have blank screen and "Bad Gateway 502"
Hassio Raspberry Pi 3b+, Microsoft Edge
Configuration of Grocy
```
culture: pl
currency: PLN
entry_page: stock
features:
  batteries: true
  calendar: true
  chores: true
  equipment: true
  recipes: true
  shoppinglist: true
  stock: true
  tasks: true
tweaks:
  chores_assignment: true
  multiple_shopping_lists: true
  stock_best_before_date_tracking: true
  stock_location_tracking: true
  stock_price_tracking: true
  stock_product_freezing: true
  stock_product_opened_tracking: true
  stock_count_opened_products_against_minimum_stock_amount: true
ssl: false
certfile: fullchain.pem
keyfile: privkey.pem
```
Logs:
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] 01-grocy: applying... 
[fix-attrs.d] 01-grocy: exited 0.
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 
-----------------------------------------------------------
 Add-on: Grocy
 ERP beyond your fridge! A groceries & household management solution for your home
-----------------------------------------------------------
 Add-on version: 0.15.0
 You are running the latest version of this add-on.
 System: Home Assistant OS 6.4  (aarch64 / raspberrypi3-64)
 Home Assistant Core: 2021.9.7
 Home Assistant Supervisor: 2021.09.6
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] grocy.sh: executing... 
[20:56:37] INFO: Patching Grocy to fix relative URL handling...
patching file views/layout/default.blade.php
Hunk #1 succeeded at 88 with fuzz 2 (offset 1 line).
[cont-init.d] grocy.sh: exited 0.
[cont-init.d] nginx.sh: executing... 
[cont-init.d] nginx.sh: exited 0.
[cont-init.d] php-fpm.sh: executing... 
[cont-init.d] php-fpm.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[20:56:41] INFO: Starting PHP-FPM...
[20:56:51] INFO: Starting NGinx....
2021/10/05 20:58:27 [error] 628#628: *1 recv() failed (104: Conne
![Bez nazwy-1](https://user-images.githubusercontent.com/87829750/136087041-91858021-71a2-4ac0-8ed8-69dbd2270ce0.png)
ction reset by peer) while reading response header from upstream, client: 172.30.32.2, server: a0d7b954-grocy, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9002", host: "smartzoli.duckdns.org:8123", referrer: "https://smartzoli.duckdns.org:8123/a0d7b954_grocy"
```
What`s is wrong with my configuration?

**Comment by TerrorBite** [comment]
See #224. Unfortunately there is a bug that occurs in the `aarch64` build of this addon which means it can't be used on Home Assistant OS installed on a Raspberry Pi 3B+. This is apparently going to be fixed in the next release of this addon.

**Comment by Musickiller** [comment]
On version 0.15.0 (which seem to be latest available on HA, the issue still persists

**Comment by BarBaar44** [comment]
Confirming this on a Odroid C2 running Hassio

**Comment by JvdBBBB** [comment]
On version 0.15.0 (which seem to be latest available on HA, the issue still persists.

**Comment by crisjff** [comment]
Fixed for me on version 0.16.0, Home Assistant OS 7.0, core-2021.12.4, supervisor-2021.12.2, Hassio Raspberry Pi 3b+. Thank you.
